CMakeLists.txt

```diff
@@ -112,9 +112,7 @@ include(autocmake_safeguards)
 find_package(PythonInterp REQUIRED)
 find_package(PythonLibs ${PYTHON_VERSION_MAJOR} REQUIRED)
 
-add_subdirectory(external)
-
-    # external projects manage their own OpenMP and c++YY flags, so only add to CXX_FLAGS for psi4_core
+# external projects manage their own OpenMP and c++YY flags, so only add to CXX_FLAGS for psi4_core
```

Comment:
The trouble comes b/c there's not much facility for _un_-setting flags, and openmp and c++11 settings aren't cmake entities, rather they're expressed through flags. If openmp is enabled in psi4, that adds `-fopenmp` or whatever to `CMAKE_C_FLAGS`. Naturally that var gets sent to libefp extproj. But libefp needs to build w/o openmp (don't ask), yet because the flag is already in `CMAKE_C_FLAGS`, passing `ENABLE_OPENMP=OFF` to the libefp extproj has no effect.